src/frontend/planner_test/tests/testdata/index_selection.yaml

```diff
@@ -85,11 +85,8 @@
     create index idx2 on t1(a) include(a);
     select count(1) from t1;
   batch_plan: |
-    BatchSimpleAgg { aggs: [sum0(count(1:Int32))] }
-    └─BatchExchange { order: [], dist: Single }
-      └─BatchSimpleAgg { aggs: [count(1:Int32)] }
-        └─BatchProject { exprs: [1:Int32] }
-          └─BatchScan { table: idx2, columns: [], distribution: SomeShard }
+    BatchSimpleAgg { aggs: [count(1:Int32)] }
```

Comment:
Is this correct? Seems like it is. The batchsimpleagg is simplified to evaluating a const expr, so it has no input depdendence, so it becomes distribution single. Great!

Reply:
I think it is not correct :hot_face: 
the `count(1)` and `count(*)` do not have input columns but it depend on the input relations, which need to give the row count of the relation.

Reply:
fixed